hack/release/push-node.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this script replaces hack/release/build/push-node.sh for Kubernetes v1.31+
```

Comment:
(it started as a copy, I added `VERSION="$1"` and then ripped out all of the kubernetes build and version detection bits in favor of passing this as the `kind build node-image` argument)